### ISSUE TITLE
Bloch sphere distortion on Jupyter Lab

### ISSUE THREAD

**Issue by RodMech** [qutip/qutip#1329]
**Describe the issue**
Bloch sphere distortion:

Hi, I am using a jupyter-lab version: `'1.1.3'` with the following packages:

matplotlib: `'3.3.0'`
qutip: `'4.5.2'`

When I plot the Bloch sphere, it shows up with a fair amount of distortion. Can anyone please point me in the right direction?

**To Reproduce**

```python
from qutip import Bloch
bloch_sphere = Bloch()
bloch_sphere.make_sphere()
```

<img width="390" alt="Captura de pantalla 2020-07-27 a las 20 01 11" src="https://user-images.githubusercontent.com/33937164/88575578-663f4f00-d044-11ea-814c-91a72a63439c.png">

**Comment by sarsid** [comment]
This seems new to Matplotlib 3.3.0. Probably due to the following change: 
`Axes3D no longer distorts the 3D plot to match the 2D aspect ratio`
documented here: 
https://matplotlib.org/devdocs/users/whats_new.html#axes3d-no-longer-distorts-the-3d-plot-to-match-the-2d-aspect-ratio

**Comment by RodMech** [comment]
@sarsid Thank you, fixed with matplotlib 3.0.0.
